### PR TITLE
Folio 1 assign projects to manager or sales

### PR DESCRIPTION
**Pull request by RashmiDholakiya**
What's this PR do?
Assign projects to sales or manager or both

Where should the reviewer start?
from ProjectManagementController, Call the view of assign project.

How should this be manually tested?
Check the checkbox to assign project.

What are the relevant tickets?
https://arsenaltech.atlassian.net/browse/FOLIO-20

### CODE REVIEW

File: app/AssignProject.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class AssignProject extends Model
```
Comment (shabbirbharmal) [inline app/AssignProject.php]: This is wrong design. Model should not be required. You should setup onetomany relationship between User and Project using pivot table. Table name should be user_projects or something like that

File: app/Http/Controllers/RoleManagementController.php [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    public function getRoles()
+    {
+        $user = User::paginate(PAGINATE_LIMIT);
+
+        return view('assign.role',compact('user'));
+    }
+
+    public function updateRole(Request $request)
+    {
+        $project = User::findOrFail($request['user_id']);
```
Comment (shabbirbharmal) [inline app/Http/Controllers/RoleManagementController.php]: User::findOrFail won't return project. So why is variable named $project?

File: app/Project.php [hunk 3]
```diff
@@ -39,6 +39,11 @@ public function timelog() {
     public function costs() {
         return $this->hasMany(ProjectCost::class)->orderBy('month_logged')->orderBy('staff_id');
     }
+
+    public function assign_projects()
```
Comment (shabbirbharmal) [inline app/Project.php]: Never use underscore for method names

File: app/AssignProject.php (hunk 1 above)
Comment (palash88) [inline app/AssignProject.php]: Discussed with Rashmi on the above code flow and she will be implementing it to User Roles Assigned to Project as compared to previously set as Role Type to Projects.

**Pull request closed without merging**